NuclearData/ceNeutronData/aceDatabase/aceNeutronDatabase_class.f90

```diff
@@ -645,6 +707,68 @@ subroutine init_urr(self)
 
   end subroutine init_urr
 
+  !!
+  !!  Checks through all nuclides, creates map with nuclides present and corresponding 0K nuclide
+  !!
+  !!  NOTE: compares the first 5 letters of the ZAID.TT. It would be wrong with isotopes
+  !!        with Z > 99
```

Comment:
I am a bit of scared of this one since some nuclides have only 4 digits in ZAID (e.g. `7014`) not that they will ever be significant for DBRC but if they are accidentally included in the list there is a chance code might do some weird things. 

Perhaps we can use the `.` as the separator and just compare the ZAID part before the first `.`?